conf/app.routes

```diff
@@ -5,4 +5,4 @@ GET           /relationships/agent/:arn/service/:service/client/:clientId
 PUT           /relationships/agent/:arn/service/:service/client/:clientId        @uk.gov.hmrc.agentfirelationship.controllers.RelationshipController.createRelationship(arn: String, service: String, clientId: String)
 DELETE        /relationships/agent/:arn/service/:service/client/:clientId        @uk.gov.hmrc.agentfirelationship.controllers.RelationshipController.deleteRelationship(arn: String, service: String, clientId: String)
 
-GET           /relationships/paye/agent/:arn/client/:clientId                    @uk.gov.hmrc.agentfirelationship.controllers.RelationshipController.afiCheckRelationship(arn: String, clientId: String)
\ No newline at end of file
+GET           /relationships/afi/agent/:arn/client/:clientId                     @uk.gov.hmrc.agentfirelationship.controllers.RelationshipController.afiCheckRelationship(arn: String, clientId: String)
```

Comment:
You may need to modify the read me too https://github.com/hmrc/agent-fi-relationship/blob/e0b16cc208a835badc70c11948669bd46131f77f/README.md#access-control---view-relationship-for-afi

Reply:
Done ;) make sure you squash